tests/draft2019-09/optional/ecmascript-regex.json

```diff
@@ -1,11 +1,29 @@
 [
     {
         "description": "ECMA 262 regex non-compliance",
-        "schema": { "format": "regex" },
+        "schema": {
+            "type": "string",
+            "pattern": "^\\S\\Z"
```

Comment:
I think this might be simpler/more readable as `"^x\\Z"` - took me a minute to recall what \S is.

Reply:
Good idea.